firmware/lib/thermistorProbe/thermistorUSP10976.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+#ifndef THERMISTOR_USP10976_H
+#define THERMISTOR_USP10976_H
```

Comment:
`#pragma once` is redundant with the header guard (`ifndef`, `define`) you have right after it (they each ensure the header file is only included once, so you only need one of them)

We've been going with header guards in all the other files, so lets stick to that for consistency.

tldr: remove `#pragma once`